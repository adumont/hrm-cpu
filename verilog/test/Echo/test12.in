59 f8
47 00
30 fb
34 6e
57 ed
35 a9
12 d1
01 e9
58 0d
11 7f
46 f8
02 0d
01 1d
55 c4
30 9e
49 0b
17 52
43 89
47 47
24 ea
29 ab
05 8a
17 bf
19 f6
05 7c
58 15
03 eb
40 1f
44 68
21 fb
07 48
33 7e
38 a2
28 08
10 ca
42 27
