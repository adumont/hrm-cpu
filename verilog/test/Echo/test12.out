f8
00
fb
6e
ed
a9
d1
e9
0d
7f
f8
0d
1d
c4
9e
0b
52
89
47
ea
ab
8a
bf
f6
7c
15
eb
1f
68
fb
48
7e
a2
08
ca
27
